examples/js/nextjs-new/README.md

```diff
@@ -0,0 +1,57 @@
+# Next.js
+
+This is an example project showing how to use the `@bugsnag/js` notifier to track server, browser, and Vercel serverless errors on a Next.js project.
```

Comment:
Would this be accurate?  Or could "server" have another meaning beyond "custom server"?
```suggestion
This is an example project showing how to use the `@bugsnag/js` notifier to track browser and Vercel serverless errors on a Next.js project. Custom servers are not supported due to the potential for data leakage between concurrent requests.
```